awx/ui/src/screens/NotificationTemplate/shared/Notifications.helptext.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { t } from '@lingui/macro';
+
+const NotificationHelpText = () => ({
```

Comment:
why a function instead of exporting the values in an object?

Reply:
Good catch....habitt